LEAF_Nexus/templates/permission_iframe.tpl

```diff
@@ -31,7 +31,7 @@
         <!--{/if}-->
     <!--{else}-->
 <div style="padding: 4px; color: green">
-    <div onkeypress="triggerClick(event, this.id)" tabindex="0" id="editpermissions" class="buttonNorm" onclick="window.open('index.php?a=view_permissions&amp;indicatorID=<!--{$indicatorID|strip_tags|escape}-->&amp;UID=<!--{$UID|strip_tags|escape}-->','Orgchart','width=840,resizable=yes,scrollbars=yes,menubar=yes').focus();">
+    <div onkeydown="triggerClick(event, this.id)" tabindex="0" id="editpermissions" class="buttonNorm" onclick="window.open('index.php?a=view_permissions&amp;indicatorID=<!--{$indicatorID|strip_tags|escape}-->&amp;UID=<!--{$UID|strip_tags|escape}-->','Orgchart','width=840,resizable=yes,scrollbars=yes,menubar=yes').focus();">
```

Comment:
Was it all wonky here when it was 'onkeypress'?

Reply:
I changed them to keydown mostly because keypress is marked as deprecated.  It didn't cause issues (yet) in most places but figured might as well get them now.